packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -127,18 +127,36 @@ export const Docs: () => JSX.Element = () => {
         Controls](./?path=/docs/composites-call-basicexample--basic-example#customize-call-controls) to learn more.
       </Description>
 
-      <Heading>Adding file sharing</Heading>
+      <Heading>Adding file sharing in ACS Chat</Heading>
       <SingleLineBetaBanner />
       <Description>
-        The CallWithChat Composite supports file sharing capabilities in conjunction with your choice of a storage
-        solution. Using the provided APIs, you can enable the composite to support uploading files and displaying them
-        on the send box before sending, and the message thread once they have been sent or received. For an end to end
-        tutorial on enabling file sharing with Azure Blob Storage as your choice of a storage solution, refer to our
-        [tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial). File
-        sharing is supported for Teams interop scenarios by default.
```

Comment:
I see we do call it out here, that developers get this implementation by default, instead of creating a new section I recommend just making it a new paragraph and summarize everything you mentioned there instead. But again I think we need to reach out to ACS Docs team to ask for their advice, we clearly have an issue with how our storybook is organized

Reply:
there's a refactor pending PM's review right now (and has been reviewed by Docs team), this PR is meant to unblock customer right now who's having issue with a broken link

Reply:
this section would likely be removed after MSDoc went live, but i have just updated my PR to combine them into one section